Assets/Scripts/Layers/LayerTypes/WMSLayerGameObject.cs

```diff
@@ -0,0 +1,67 @@
+using Netherlands3D.Twin.Layers.Properties;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Netherlands3D.Twin.Layers
+{
+    /// <summary>
+    /// Extention of LayerGameObject that injects a 'streaming' dataprovider WMSTileDataLayer
+    /// </summary>
+    public class WMSLayerGameObject : CartesianTileLayerGameObject, ILayerWithPropertyData
+    {
+        private WMSTileDataLayer wmsProjectionLayer;
+        public WMSTileDataLayer WMSProjectionLayer
+        {
+            get
+            {
+                if (wmsProjectionLayer == null)
+                    wmsProjectionLayer = GetComponent<WMSTileDataLayer>();
+                return wmsProjectionLayer;
+            }
+        }
+
+        protected LayerURLPropertyData urlPropertyData = new();
```

Comment:
Kan het private zijn? Een field zou alleen protected, internal of public moeten zijn als er een reden voor is

Reply:
zal het omschrijven, maar het lijkt me wel dat je met elke ervende vorm van een wmslayergameobject de url propertydata wil kunnen accessen